funcx_sdk/funcx/serialize/concretes.py

```diff
@@ -134,6 +137,77 @@ def deserialize(self, payload):
         return function
 
 
+class CombinedCode(SerializeBase):
+    """This method uses multiple methods to serialize a function
+
+    # These should be a superset of the methods
+    Code from interpreter/main        : Yes
+    Code from notebooks               : Yes
+    Works with mismatching py versions: Yes
+    Decorated fns                     : Yes
+    """
+
+    identifier = "10\n"
+
+    def get_multiple_payloads(self, payload):
```

Comment:
Please consider adding typing information.

Reply:
I did not add typing previously as the original file did not have it, adding typing to facade.py and concretes.py should prob be a separate PR. However, I did add it to this file for now.